job_definitions/code_backups.yml

```diff
@@ -0,0 +1,65 @@
+{% set packages = [
+    "digitalmarketplace-api",
+    "digitalmarketplace-search-api",
+    "digitalmarketplace-admin-frontend",
+    "digitalmarketplace-agreements",
+    "digitalmarketplace-antivirus-api",
+    "digitalmarketplace-api-docs",
+    "digitalmarketplace-apiclient",
+    "digitalmarketplace-aws",
+    "digitalmarketplace-bad-words",
+    "digitalmarketplace-brief-responses-frontend",
+    "digitalmarketplace-briefs-frontend",
+    "digitalmarketplace-buyer-frontend",
+    "digitalmarketplace-cloudwatch-to-graphite",
+    "digitalmarketplace-content-loader",
+    "digitalmarketplace-credentials",
+    "digitalmarketplace-deployment",
+    "digitalmarketplace-docker-base",
+    "digitalmarketplace-framework-application-guidance",
+    "digitalmarketplace-frameworks",
+    "digitalmarketplace-frontend-toolkit",
+    "digitalmarketplace-functional-tests",
+    "digitalmarketplace-g-cloud-service-submission",
```

Comment:
Probably don't need Jenkins jobs for archived repos...

Reply:
Done 👍